src/tr/merlinscans/src/eu/kanade/tachiyomi/extension/tr/merlinscans/MerlinScans.kt

```diff
@@ -1,12 +1,457 @@
+        return SManga.create().apply {
+            title = document.select("h1.series-title").text()
+            thumbnail_url = document.select(".cover-image").attr("src")
+            description = document.select(".description-text").text()
+
+            author = document.select(".metadata-item")
+                .find { it.select(".metadata-icon-label").text().contains("Yazar") }
+                ?.select(".metadata-value")?.text()
+
+            val statusText = document.select(".metadata-item")
+                .find { it.select(".metadata-icon-label").text().contains("Durum") }
+                ?.select(".metadata-value")?.text() ?: ""
+
+            status = when {
+                statusText.contains("Devam Ediyor", true) -> SManga.ONGOING
+                statusText.contains("Tamamlandı", true) -> SManga.COMPLETED
+                statusText.contains("Ara Verildi", true) -> SManga.ON_HIATUS
+                statusText.contains("İptal", true) -> SManga.CANCELLED
+                else -> SManga.UNKNOWN
+            }
+
+            genre = document.select(".category-badge")
+                .joinToString { it.text() }
+        }
+    }
+
+    override fun chapterListRequest(manga: SManga): Request {
+        val normalizedUrl = normalizeUrl(manga.url)
+        return GET(baseUrl + normalizedUrl, headers)
+    }
+
+    override fun chapterListSelector() = ".chapters-grid .chapter-item"
+
+    override fun chapterFromElement(element: Element): SChapter {
+        return SChapter.create().apply {
+            setUrlWithoutDomain(element.attr("href"))
+            name = element.select(".chapter-title").text()
+
+            val dateText = element.select(".chapter-date").text()
+            date_upload = parseDate(dateText)
+        }
+    }
+
+    private fun parseDate(dateStr: String): Long {
+        return try {
+            when {
+                dateStr.contains("önce") -> {
+                    val now = System.currentTimeMillis()
+                    when {
+                        dateStr.contains("dakika") -> {
+                            val minutes = dateStr.filter { it.isDigit() }.toLongOrNull() ?: 0
+                            now - (minutes * 60 * 1000)
+                        }
+
+                        dateStr.contains("saat") -> {
+                            val hours = dateStr.filter { it.isDigit() }.toLongOrNull() ?: 0
+                            now - (hours * 60 * 60 * 1000)
+                        }
+
+                        dateStr.contains("gün") -> {
+                            val days = dateStr.filter { it.isDigit() }.toLongOrNull() ?: 0
+                            now - (days * 24 * 60 * 60 * 1000)
+                        }
+
+                        else -> now
+                    }
+                }
+
+                else -> {
+                    SimpleDateFormat("dd.MM.yyyy", Locale.getDefault()).parse(dateStr)?.time ?: 0L
+                }
+            }
+        } catch (e: Exception) {
```

Comment:
You can do this if you don't use the exception
```suggestion
        } catch (_: Exception) {
```